merlin/models/utils/ci_utils.py

```diff
@@ -25,7 +25,7 @@
     "lightfm": "lightfm",
     "xgb": "xgboost",
 }
-OTHER_MARKERS = {"unit", "integration", "example", "datasets", "horovod", "transformers"}
+OTHER_MARKERS = {"unit", "integration", "examples", "datasets", "horovod", "transformers"}
```

Comment:
I was wondering why the examples tests were running in the main tensorflow test job. Looks like this is why, because the tests directory name is `../examples/...`, and we were looking for `example` so didn't tag the tests with the corresponding marker.